### ISSUE TITLE
Feature Request: Pull from YouTube playlist

### ISSUE THREAD

**Issue by DrFlytrap** [Qolors/FeedCord#20]
Thank you for this awesome tool. There is a channel that does a weekly news segment and puts it in a separate playlist. I would like a feed of just this playlist and not the entire channel. Is this possible? Thanks either way. Cheers!

**Comment by Qolors** [comment]
Hey there - appreciate it! Try putting this url format in to your **RSS Urls**:

- `https://www.youtube.com/feeds/videos.xml?playlist_id=YOUR_YOUTUBE_ID_HERE`

You can get the Playlist Id by clicking on the playlist in Youtube, then up in your browser search bar you'll see the url like:
- `https://www.youtube.com/watch?v=2ciHixbc4HE&list=PL0vfts4VzfNj28Xp5kmNAdIqABNGhTzKF`

The ID is the tail end after `list=`

Let me know whether this solves your issue or not

**Comment by DrFlytrap** [comment]
Ahh I did try that, but  was putting it under YouTube URLS. That seems to work now. Thank you!

I am getting a new error today on all of my YouTube URLS. Example:
W: Failed to fetch or process the RSS feed from https://www.youtube.com/@Prusa3D.. Error Count: 1

I have attached the log and my appsettings.json. It was working at first, and posted some links to the Discord.

Thanks in advance for any guidance.

[FeedCord Log 2024-03-15-001.txt](https://github.com/Qolors/FeedCord/files/14620690/FeedCord.Log.2024-03-15-001.txt)
[appsettings.json](https://github.com/Qolors/FeedCord/files/14620691/appsettings.json)
